home/web/client/actions/dapps.js

```diff
@@ -0,0 +1,5 @@
+import { createAction } from 'redux-actions';
+
+export const error = createAction('error dapps');
```

Comment:
so just in rpc we have a special case with one error handler and no `start` action?